### ISSUE TITLE
Image processing libraries rely on MediaGallery being installed

### ISSUE THREAD

**Issue by leegarner** [glFusion/glfusion#330]
$_MG_CONF is referenced by ImageMagick and GraphicsMagick:
```
mglib]# grep -rn MG_CONF *
gm_image.php:84:    global $_CONF, $_MG_CONF;
gm_image.php:86:    $JpegQuality = $_MG_CONF['jpg_quality'];
gm_image.php:206:    global $_CONF, $_MG_CONF;
gm_image.php:210:    if ($_MG_CONF['verbose']) {
gm_image.php:244:    if (($mimeType != 'image/gif') && ($_MG_CONF['jhead_enabled'] == 1)) {
gm_image.php:245:        UTL_execWrapper('"' . $_MG_CONF['jhead_path'] . "/jhead" . '"' . " -v -te " . $srcImage . " " . $destImage);
im_image.php:84:    global $_CONF, $_MG_CONF;
im_image.php:99:    $JpegQuality = $_MG_CONF['jpg_quality'];
im_image.php:227:    global $_CONF, $_MG_CONF;
im_image.php:242:    if ($_MG_CONF['verbose']) {
im_image.php:277:    if (($mimeType != 'image/gif') && ($_MG_CONF['jhead_enabled'] == 1)) {
im_image.php:278:        UTL_execWrapper('"' . $_MG_CONF['jhead_path'] . "/jhead" . '"' . " -v -te " . $srcImage . " " . $destImage);
lib-image.php:317:    global $_MG_CONF, $_CONF;
lib-image.php:326:    if ( !in_array($mimeType,$_MG_CONF['watermark_types']) ) {
```
This is from glFusion 1.8.0 dev, also observed in 1.7.3 and 1.6.6. Imaging resizing fails if $_MG_CONF['jpg_quality'] is not available. Suggest moving these required configs to the global "Images" configuration section.

**Comment by leegarner** [comment]
These changes appear to work, there are already $_CONF vars for jpg_orig_quality and path_to_jhead. Could use debug_image_upload in place of "verbose", or perhaps a new config var.
```
--- a/private/system/imglib/gm_image.php
+++ b/private/system/imglib/gm_image.php
@@ -81,9 +81,9 @@ function _img_RotateImage($srcImage, $direction,$mimeType)

 function _img_resizeImage($srcImage, $destImage, $sImageHeight, $sImageWidth, $dImageHeight, $dImageWidth, $mimeType)
 {
-    global $_CONF, $_MG_CONF;
+    global $_CONF;

-    $JpegQuality = $_MG_CONF['jpg_quality'];
+    $JpegQuality = empty($_CONF['jpg_orig_quality']) ? 85 : (int)$_CONF['jpg_orig_quality'];

     if ( $_CONF['debug_image_upload'] ) {
         COM_errorLog("_img_resizeImage: Resizing using GraphicsMagick src = " . $srcImage . " mimetype = " . $mimeType);
@@ -203,11 +203,11 @@ function _img_watermarkImage($origImage, $watermarkImage, $opacity, $location, $

 function _img_squareThumbnail($srcImage, $destImage, $sImageHeight, $sImageWidth, $dSize, $mimeType)
 {
-    global $_CONF, $_MG_CONF;
+    global $_CONF;

     $opt = '-quality ' . 91;

-    if ($_MG_CONF['verbose']) {
+    if ($_CONF['debug_image_upload']) {
         $opt .= ' -verbose';
         COM_errorLog("_img_squareThumbnail: Resizing using GraphicsMagick src = " . $srcImage . " mimetype = " . $mimeType);
     }
@@ -241,9 +241,9 @@ function _img_squareThumbnail($srcImage, $destImage, $sImageHeight, $sImageWidth
         COM_errorLog("_img_resizeImage_crop: Error - Unable to resize image - GraphicsMagick convert failed.");
         return array(false, 'Error - Unable to resize image (square thumbnail) - GraphicsMagick convert failed.');
     }
-    if (($mimeType != 'image/gif') && ($_MG_CONF['jhead_enabled'] == 1)) {
-        UTL_execWrapper('"' . $_MG_CONF['jhead_path'] . "/jhead" . '"' . " -v -te " . $srcImage . " " . $destImage);
+    if (($mimeType != 'image/gif') && ($_CONF['jhead_enabled'] == 1)) {
+        UTL_execWrapper('"' . $_CONF['path_to_jhead'] . "/jhead" . '"' . " -v -te " . $srcImage . " " . $destImage);
     }
     return array(true, '');
 }
```

**Comment by mark0263** [comment]
Geez - I went to all the effort to create the new vars and never actually consumed them...Good catch - I'll apply the fixes!

**Comment by mark0263** [comment]
Fixed with commit 13076c481bce60b380a8a97efba953b005df0b66
